condominium_manager_server/settings.py

```diff
@@ -29,6 +28,7 @@
 
 ALLOWED_HOSTS = []
 
+SECRET_JWT = os.environ.get('SECRET_JWT', 'secret_for_jwt')
```

Comment:
**Suggestion:** Instead of hardcoding the JWT secret, it would be better to fetch it from environment variables. If it's not found, raise an error instead of setting a default value. This will prevent the application from running with insecure settings.
```suggestion
SECRET_JWT = os.environ.get('SECRET_JWT')
if SECRET_JWT is None:
    raise ValueError("SECRET_JWT environment variable is not set")
```